tests/components/generic/test_camera.py

```diff
@@ -515,3 +517,22 @@ async def test_no_still_image_url(hass, hass_client):
         mock_stream.async_get_image.assert_called_once()
         assert resp.status == HTTPStatus.OK
         assert await resp.read() == b"stream_keyframe_image"
+
+
+def test_frame_interval_property(hass):
+    """Test that the frame interval is calculated and returned correctly."""
+    cam = GenericCamera(
```

Comment:
Please set up the integration instead as in the other tests here, and patch `async_get_still_stream` and assert the call args to the mock after calling `async_get_mjpeg_stream`.

Similar to this test but more simple as we don't need to patch in any callback:
https://github.com/home-assistant/core/blob/d0578509710ac27a20d1e5576e7eee15d140057d/tests/components/hyperion/test_camera.py#L129-L163

Reply:
@MartinHjelmare Done now.